Tests/test_cards.gd

```diff
@@ -0,0 +1,99 @@
+extends GutTest
+## Tests for cards
+
+
+var _battler_scene: PackedScene = load("res://Core/Battler.tscn")
+var _card_container_scene: PackedScene = load("res://Cards/CardContainer.tscn")
+var _battler: Battler = null
+var _card_container: CardContainer = null
+var _cards_in_hand: Array[CardWorld] = []
+
+
+func before_each():
+	_battler = _battler_scene.instantiate()
+	_card_container = _card_container_scene.instantiate()
+	
+	# fill deck with 50 default cards
+	_card_container.default_deck.resize(50)
+	_card_container.default_deck.fill(CardBase.new())
+	_card_container.max_hand_size = 10
+	_card_container.starting_hand_size = 5
+	
+	get_tree().root.add_child(_battler)
+	get_tree().root.add_child(_card_container)
+
+
+func after_each():
+	_battler.queue_free()
+	_card_container.queue_free()
+
+
+func test_draw_cards():
+	_card_container.draw_cards(5)
+	
+	assert_eq(_card_container.cards_in_hand.size(), 5)
+	assert_eq(_card_container.draw_pile.size(), 45)
+	assert_eq(_card_container.discard_pile.size(), 0)
+
+
+func test_draw_two_cards():
```

Comment:
would rename to `test_draw_card_2()` to not be confused with the action of drawing two cards